src/pages/MapPage.css

```diff
@@ -1,16 +1,125 @@
+/* Base styles */
+body {
+  background-color: #121212;
+  color: #ffffff;
```

Comment:
Your change here practically and effectively got rid of all our original light mode / dark mode changes, and is not good practice.